### PR TITLE
[14.0] shopinvader: forward port all recent goodies from v13 (part 2)

### PR DESCRIPTION
**Pull request by simahawk**
I split the 2nd set of commits from #970 to ease review and merging

### CODE REVIEW

**Comment by simahawk** [comment]
@PierrickBrun @hparfr could you review this?

**Review by PierrickBrun — APPROVED** [review]

File: shopinvader/wizards/shopinvader_variant_binding_wizard.py [hunk 1]
```diff
@@ -27,6 +27,7 @@ class ShopinvaderVariantBindingWizard(models.TransientModel):
         help="List of langs for which a binding must exists. If not "
         "specified, the list of langs defined on the backend is used.",
     )
+    run_immediately = fields.Boolean(help="Do not schedule jobs.")
```
Comment (hparfr) [inline shopinvader/wizards/shopinvader_variant_binding_wizard.py:30]: There is no label in the view

**Comment by hparfr** [comment]
The unbinding wiz still do not work with categories nor products.

EDIT: got it, will provide a fix

**Comment by simahawk** [comment]
> The unbinding wiz still do not work with categories nor products.
> 
> EDIT: got it, will provide a fix

I did not touch categories, what's the problem w/ products?

**Comment by hparfr** [comment]
> > The unbinding wiz still do not work with categories nor products.
> > EDIT: got it, will provide a fix
> 
> I did not touch categories, what's the problem w/ products?

When I open the wizard, I get 
```
File "/odoo/src/odoo/models.py", line 6148, in onchange
    if cmd[0] in (1, 4):
Exception

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "/odoo/src/odoo/http.py", line 639, in _handle_exception
    return super(JsonRequest, self)._handle_exception(exception)
  File "/odoo/src/odoo/http.py", line 315, in _handle_exception
    raise exception.with_traceback(None) from new_cause
TypeError: 'int' object is not subscriptable

```

Fixed in https://github.com/shopinvader/odoo-shopinvader/pull/990

PS: probem was existing before your pr.

**Pull request merged**